### PR TITLE
Ejercicio nested resources

### PR DESCRIPTION
**Pull request by EmilioCristalli**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
-  resources :boards
+
+  resources :boards do
+    resources :lists, only: [:index, :new, :create]
```
Comment (EmilioCristalli) [inline config/routes.rb]: I wonder if `lists#index` should be `boards#show`

File: app/controllers/lists_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  def new
+    @list = List.new
+  end
+
+  def create
+    @list = @board.lists.build(list_params)
+
+    respond_to do |format|
+      if @list.save
+        format.html { redirect_to board_lists_path(@board), notice: 'Board was successfully created.' }
```
Comment (EmilioCristalli) [inline app/controllers/lists_controller.rb]: is the `flas[:notice]` being shown in `board_lists_path`? if not, should we set it here?

File: app/controllers/lists_controller.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        format.json { render :show, status: :created, location: @list }
```
Comment (EmilioCristalli) [inline app/controllers/lists_controller.rb]: should we add JSON responses?

**Comment by EmilioCristalli** [comment]
a couple questions but looks good to me

File: app/controllers/lists_controller.rb (hunk 2 above)
Comment (jmbejar) [inline app/controllers/lists_controller.rb]: Let's remove it 👍

File: app/controllers/lists_controller.rb (hunk 3 above)
Comment (jmbejar) [inline app/controllers/lists_controller.rb]: No needed. I will simplify.

**Pull request merged**